src/components/common/buttons/CustomIcon.js

```diff
@@ -5,16 +5,18 @@ import Icon from '../view/Icon'
 import { withStyles } from '../../../lib/styles'
 
 const CustomIcon = ({ styles, theme, name, color, size, circleSize = 16, wrapper = true, reverse, reverseColor }) => {
-  const borderRadius = useMemo(() => circleSize / 2, [circleSize])
+  const borderRadius = useMemo(() => (wrapper ? circleSize / 2 : null), [circleSize, wrapper])
   return (
     <View
       style={[
         styles.imageIcon,
         {
+          backgroundColor: reverse ? color : reverseColor,
```

Comment:
@roma-hladilka border radius was **always** set e.g.

```javascript
const CustomIcon = ({ styles, theme, name, color, size, circleSize = 16, wrapper = true, reverse, reverseColor }) => {
  const borderRadius = useMemo(() => (wrapper ? circleSize / 2 : null), [circleSize, wrapper])
  
  return (
    <View
      style={[
        styles.imageIcon,
        {
          borderRadius,
          backgroundColor: reverse ? color : reverseColor,
        },
        wrapper && {
          width: circleSize,
          height: circleSize,
        },
      ]}
    >
      <Icon name={name} size={size} color={theme.colors.surface} />
    </View>
  )
}
```

Reply:
if it doesn't needs to be set, we do not need wrapper prop at all, just set circleSize to false value where we had been setting wrapper = false

e.g.

```javascript
const CustomIcon = ({ styles, theme, name, color, size, circle = 16, reverse, reverseColor }) => {
  const [_circle, borderRadius] = useMemo(() => {
    const _circle = circle !== false

    return [_circle, _circle ? circle / 2 : null]
  }, [circle, wrapper])
  
  return (
    <View
      style={[
        styles.imageIcon,
        {
          backgroundColor: reverse ? color : reverseColor,
        },
        _circle && {
          borderRadius,
          width: circle,
          height: circle,
        },
      ]}
    >
      <Icon name={name} size={size} color={theme.colors.surface} />
    </View>
  )
}

// ...

<CustomIcon circle={false} ... />
```